orttraining/orttraining/models/gpt2/main.cc

```diff
@@ -119,7 +121,11 @@ Status ParseArguments(int argc, char* argv[], GPT2Parameters& params, OrtParamet
 
     params.num_train_steps = flags["num_train_steps"].as<int>();
     params.batch_size = flags["train_batch_size"].as<int>();
-
+    if (flags.count("eval_batch_size")) {
```

Comment:
didn't see a "eval_batch_size" in the options above. should this just be directly set to params.batch_size?

Reply:
will check